services/src/error.rs

```diff
@@ -133,6 +133,7 @@ pub enum Error {
 
     NoWorkflowForGivenId,
 
+    #[snafu(display("Database error: {source}"))]
```

Comment:
do we really output the db internas to the user?

Reply:
was only for debugging c7e5c8ae4b1a41260f1e2d545f57f2baf849fac9